apps/api/src/campaign/campaign.service.ts

```diff
@@ -20,11 +20,29 @@ export class CampaignService {
   constructor(private prisma: PrismaService) {}
 
   async listCampaigns(): Promise<Campaign[]> {
-    return this.prisma.campaign.findMany({
+    const campaigns = await this.prisma.campaign.findMany({
       include: {
-        summary: { select: { reachedAmount: true } },
-      },
+        vaults: {
+          select: {
+            donations: { select: { amount: true} }
```

Comment:
I wonder if we return all donation records to the frontend as well as the other data from the campaign object

Can you post the whole response?

Reply:
yep, indeed vaults and donations get returned to frontend, the TODO on the lines below was for that.
fixed: by setting vaults to [] removes these unnecessary records